application/src/main/java/uk/nhs/hee/tis/revalidation/service/RecommendationElasticSearchService.java

```diff
@@ -120,4 +131,14 @@ private Iterable<RecommendationView> findRecommendationViewsByGmcNumber(
     }
     return result;
   }
+
+  private String getAutocompleteFieldValue(String fieldName, RecommendationView result) {
```

Comment:
Just IMO, this function is much more generic.  `getFieldValueAsString` seems like a more fitting description.